frontend/packages/kubevirt-plugin/src/models.ts

```diff
@@ -52,10 +53,9 @@ export const VirtualMachineInstanceReplicaSetModel: K8sKind = {
   id: 'virtualmachineinstancereplicaset',
 };
 
-/* TODO(mlibra): migrate templates
 export const VmTemplateModel: K8sKind = {
```

Comment:
You shouldn't override the existing template model. It's already defined.

Reply:
@spadgett Thanks, this comment should be in:
https://github.com/openshift/console/pull/1673/files#diff-8066fd03cba594c7f8fb2be0890b95f9L55

This PR requires #1673, and this line is actually from the other PR.

I will adress this issue in #1673 :+1: